### PR TITLE
OTP voice support

### PR DESCRIPTION
**Pull request by dorsha**
## Related Issues
https://github.com/descope/etc/issues/2668

## Description
OTP voice support

## Must
- [x] Tests
- [ ] Documentation (if applicable)

### CODE REVIEW

File: descope/sdk/auth.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ type OTP interface {
 	UpdateUserEmail(ctx context.Context, loginID, email string, updateOptions *descope.UpdateOptions, request *http.Request) (maskedAddress string, err error)
 
 	// UpdateUserPhone - Use to update phone and validate via OTP
-	// allowed methods are phone based methods - whatsapp and SMS
+	// allowed methods are phone based methods - whatsapp and SMS or Voice call
```
Comment (aviadl) [inline descope/sdk/auth.go:112]: Whatsapp, SMS or ...

File: descope/sdk/auth.go [hunk 2]
```diff
@@ -152,7 +152,7 @@ type Password interface {
 	// redirectURL is an optional parameter that is used by Magic Link or Enchanted Link
 	// if those are the chosen reset methods. See the Magic Link and Enchanted Link sections
 	// for more details.
-	// templateOptions is used to pass dynamic options for the messaging (email / text message) template
+	// templateOptions is used to pass dynamic options for the messaging (email / text message, voice call) template
```
Comment (aviadl) [inline descope/sdk/auth.go:155]: user `/` to be consistent with the previous part of the sentence

File: descope/sdk/auth.go (hunk 1 above)
Comment (dorsha) [reply descope/sdk/auth.go:112]: done

File: descope/sdk/auth.go (hunk 2 above)
Comment (dorsha) [reply descope/sdk/auth.go]: done

**Review by aviadl — APPROVED** [review]

**Pull request merged**